src/data/communities.json

```diff
@@ -110,6 +110,16 @@
     "telegram": "https://t.me/FFDGCHENNAI"
   },
   {
+    "name": "CTOtalk",
```

Comment:
Hey @selvasudar bro this looks like a conference and a platform where different speakers present, 
Can I please know if this would be a tech community similar to the likes of fosschennai and chennai py or this is something different. 

Would love to know more about this. 

Thanks a lot 

Cheers :)